src/base/index.vue

```diff
@@ -990,15 +1181,15 @@ export default {
           type: view.type,
           linkedTo: view.linkedTo,
         };
-        if (view.settings) {
-          refactorView.options = JSON.stringify(view.settings);
-        }
-        if (!view.settings && view.type === "configurable") {
+        if (view.settings && view.type === "configurable") {
+          refactorView.name = view.nameID;
           const options = [];
-          view.renderData.map((rData) => {
+          view.render_data.map((rData) => {
```

Comment:
I don't understand why have you changed the field name from renderData to render_data to end up having to update it all over the code ?

Reply:
Sometimes they are returned as render_data and in other cases as renderData so I had to make this change

Reply:
please define "sometimes" are you sure you aren't breaking for the cases when it's rendered as renderData?